terraform/datadog/etcd.tf

```diff
@@ -0,0 +1,23 @@
+resource "datadog_monitor" "etcd_one_leader" {
+  name                = "${format("%s etcd IsLeader count is exactly one (%s)", var.env, element(list("upper", "lower"), count.index))}"
+  type                = "query alert"
+  message             = "There is not exactly one etcd reporting IsLeader."
+  escalation_message  = "There is still not exactly one etcd reporting IsLeader."
+  notify_no_data      = false
+  require_full_window = true
+
+  count = 2
+
+  query = "${
+    format(
+      "min(last_5m):sum:cf.etcd.IsLeader{deployment:%s}.fill(last, 60) %s",
+      var.env,
+      element(list("> 1", "< 1"), count.index)
+    )}"
+
+  thresholds {
+    critical = "${element(list("1", "1"), count.index)}"
```

Comment:
e.g:
```
diff --git a/terraform/datadog/tps.tf b/terraform/datadog/tps.tf
index eef05b6..6eaefb8 100644
--- a/terraform/datadog/tps.tf
+++ b/terraform/datadog/tps.tf
@@ -10,13 +10,13 @@ resource "datadog_monitor" "tps_watcher_lock_held_once" {
 
   query = "${
     format(
-      "min(last_5m):sum:cf.tps_watcher.LockHeld.v1_locks_tps_watcher_lock{deployment:%s}.fill(last, 60) %s",
+      "avg(last_5m):sum:cf.tps_watcher.LockHeld.v1_locks_tps_watcher_lock{deployment:%s}.fill(last, 60) %s",
       var.env,
-      element(list("> 1", "< 1"), count.index)
+      element(list("> 0.8", "< 1.2"), count.index)
     )}"
 
   thresholds {
-    critical = "1" # This value must match the threshold set in the query. Otherwise datadog API would fail obscurely
+    critical = "${element(list("0.8", "1.2"), count.index)}"
   }
 
   tags = ["deployment:${var.env}", "service:${var.env}_monitors", "job:tps_watcher"]
```